views/v_ksm_data_mart.sql

```diff
@@ -309,6 +316,8 @@ Select
   , degrees.date_added
   , degrees.date_modified
   , degrees.operator_name
+  , case when deg.FIRST_KSM_YEAR is not null and degrees.school_code = 'KSM' then deg.FIRST_KSM_YEAR else '' End As First_KSM_Year
+  , case when deg.FIRST_MASTERS_YEAR is not null and degrees.school_code = 'KSM' then deg.FIRST_MASTERS_YEAR Else '' End as First_KSM_Masters_Year
```

Comment:
Remove else '' from here and above - this will keep year as numeric, right now it's forced to string